src/array/cpu/spmm.h

```diff
@@ -2,6 +2,10 @@
  *  Copyright (c) 2020 by Contributors
  * \file array/cpu/spmm.h
  * \brief SPMM CPU kernel function header.
+ * \author Sanchit Misra <sanchit.misra@intel.com>,
+ *         Ramanarayan Mohanty <ramanarayan.mohanty@intel.com>,
+ *         Vasimuddin Md <vasimuddin.md@intel.com>,
+ *         Sasikanth Avancha <sasikanth.avancha@intel.com>
```

Comment:
it's better to add the original authors as well.

Reply:
actually, i think it's better to move the block spmm code to another header file. the code will also look cleaner.

Reply:
Absolutely. I didn't put their names only because I didn't know the name of the original authors of this file. But I like your suggestion of moving block spmm code to another header file. I will do that. Should that header file be in src/array/cpu/ ? How about src/array/cpu/spmm_blocking.h?

Reply:
`src/array/cpu/spmm_blocking.h` sounds good to me.